# Timestamp header
time stamp

# Current left velocity (m/s)
float32 left_velocity

# Current right velocity (m/s)
float32 right_velocity
